packages/itwinui-css/src/button/classes.scss

```diff
@@ -4,67 +4,54 @@
 
 .iui-button {
   @include iui-button;
-  @include iui-button-size;
-
-  &.iui-active {
-    @include iui-button-active;
-  }
-
-  &.iui-default {
-    @include iui-button-default;
-  }
+}
 
-  &.iui-high-visibility {
-    @include iui-button-high-visibility;
-  }
+.iui-button-label {
+  @include iui-button-label;
+}
 
-  &.iui-cta {
-    @include iui-button-cta;
-  }
+.iui-button-icon {
+  @include iui-button-icon;
+}
 
-  &.iui-idea {
-    @include iui-button-idea;
-  }
+// #region Button variants
+[data-iui-button-variant='default'] {
+  @include iui-button-default;
+}
 
-  &.iui-borderless {
-    @include iui-button-borderless;
-  }
+[data-iui-button-variant='borderless'] {
+  @include iui-button-borderless;
+}
 
-  @include iui-button-size(medium);
+[data-iui-button-variant='high-visibility'] {
+  @include iui-button-status-color(primary);
+}
 
-  &.iui-small {
-    @include iui-button-size(small);
+[data-iui-button-variant='cta'] {
+  @include iui-button-status-color(positive);
+}
 
-    &.iui-borderless {
-      @include iui-button-size(small, borderless);
-    }
-  }
+[data-iui-button-variant='idea'] {
+  @include iui-button-idea;
+}
+// #endregion Button variants
 
-  &.iui-large {
-    @include iui-button-size(large);
+.iui-button-dropdown {
+  @include iui-button-dropdown;
+}
 
-    &.iui-borderless {
-      @include iui-button-size(large, borderless);
-    }
-  }
+.iui-button-split {
+  @include iui-button-split;
 }
 
 .iui-button-group {
   @include iui-button-group;
-
-  &-vertical {
-    @include iui-button-group(vertical);
-  }
-
-  &-overflow-x {
-    @include iui-button-group-overflow-x;
-  }
 }
 
-.iui-button-split-menu {
-  @include iui-button-split-menu;
+.iui-button-group-vertical {
+  @include iui-button-group(vertical);
 }
 
-.iui-button-icon {
-  @include iui-button-icon;
+.iui-button-group-overflow-x {
```

Comment:
Why changed this?

Reply:
We've been trying to avoid `&-` in a bunch of our CSS 1.0 PRs so that it's easier to search & find classes.

If I search for `.iui-button-group-overflow-x` it won't find this:
```scss
.iui-button-group {
  &-overflow-x { … }
}
```